sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/_azure_appconfiguration_client.py

```diff
@@ -107,8 +123,37 @@ def _create_appconfig_pipeline(self, **kwargs):
                 self._config.headers_policy,
                 self._config.user_agent_policy,
                 AppConfigRequestsCredentialsPolicy(self._config.credentials),
+                self._config.retry_policy,
                 SyncTokenPolicy(),
+                self._config.logging_policy,  # HTTP request/response log
+                DistributedTracingPolicy(**kwargs),
+                HttpLoggingPolicy(**kwargs)
+            ]
+
+        if not transport:
+            transport = RequestsTransport(**kwargs)
+
+        return Pipeline(transport, policies)
+
+    def _create_appconfig_aad_pipeline(self, base_url, credential, **kwargs):
+        transport = kwargs.get('transport')
+        policies = kwargs.get('policies')
+        if base_url.endswith('/'):
+            scope = base_url + '.default'
+        else:
+            scope = base_url + '/.default'
+
+        if policies is None:  # [] is a valid policy list
```

Comment:
Not a huge fan of duplicating the list of policies twice, with only one diff on the credential policy.
If tomorrow we add a new type of policy for some reason, the likeliness of forgetting to update one of the two lists is real.
Also, we should take advantage of the fact that the pipeline skip policies that are None.

I'd rather have one method, with some code like:
```python
policies = [
  self._config.headers_policy,
  self._config.user_agent_policy,
  AppConfigRequestsCredentialsPolicy(self._config.credentials) if aad_mode else None
  self._config.retry_policy,
  SyncTokenPolicy(),
  self._config.logging_policy,  # HTTP request/response log
  DistributedTracingPolicy(**kwargs),
  HttpLoggingPolicy(**kwargs)
]
```